otherlibs/stdlib_alpha/effect.ml

```diff
@@ -392,8 +392,14 @@ type ('a, 'e) perform = ('a, 'e) op * 'e Handler.t'
 
 external perform : ('a, 'e) perform -> 'a = "%perform"
 
+(* Stacks are represented as tagged pointers, and garbage collection
```

Comment:
It's not _quite_ true that GC invalidates them, just that they do not keep their contents alive, so if they are the only remaining reference they will be invalidated. This is the case when we allocate a fresh stack or swap a stack out of a cont.